src-tauri/Cargo.toml

```diff
@@ -20,7 +20,7 @@ serde = { version = "1.0", features = ["derive"] }
 tauri = { version = "1.1.1", features = ["api-all", "devtools"] }
 tauri-plugin-localhost = "0.1.0"
 portpicker = "0.1"
-lodestone_core = {git = "https://github.com/Lodestone-Team/lodestone_core", branch = "main"}
+lodestone_core = {git = "https://github.com/Lodestone-Team/lodestone_core", branch = "dev/refactor-setup"}
```

Comment:
We should probably change this to release/0.4.3, and backend should merge dev/refactor-setup in to 0.4.3 @CheatCod